src/scss/_components.diagram.chart.scss

```diff
@@ -15,29 +10,56 @@ $circle-height: 28px;
   &__list {
     max-width: 70rem;
     margin: 0 auto;
-    display: grid;
-    grid-row-gap: 2rem;
-    grid-column-gap: 0.5rem;
     text-align: center;
-    grid-template-columns: repeat(var(--count), 1fr);
-    grid-template-rows: calc(var(--height) + #{2*$circle-height}) auto;
-    overflow: hidden;
+    // overflow: hidden;
     position: relative;
+    display: flex;
+    justify-content: space-between;
 
-    @media (min-width: $bp-lg) {
-      grid-column-gap: 1rem;
+    @supports (display: grid) {
+      display: grid;
+      grid-row-gap: 2rem;
+      grid-column-gap: 0.5rem;
+      grid-template-columns: repeat(6, 1fr);
+      align-items: flex-start;
+
+      @media (min-width: $bp-lg) {
+        grid-column-gap: 1rem;
+      }
+    }
+  }
+
+  &__item {
+    width: 10.75rem;
+    height: 35rem;
+    position: relative;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-content: flex-end;
+
+    @supports (display: grid) {
+      display: grid;
+      grid-row-gap: 2rem;
+      grid-template-rows: 28.5rem auto;
+      height: unset;
+      width: unset;
     }
   }
 
   &__platform {
     perspective: 800px;
     position: absolute;
-    top: calc(var(--height) + #{2*$circle-height} - 2.75rem);
+    top: calc(25rem + 10px - 2.75rem);
```

Comment:
Is the JS calculating and overriding this?

Reply:
I don't have any JS set up for this. Could you clarify what you mean?